recipes/conda-tree/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "conda-tree" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rvalieris/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0a2adacc7416d02b6a8a923bc23762655d0c28dd4f0498f3cd72ba3bffc4a1e7
+
+build:
+  number: 0
+  noarch: python
+  script:    # See https://github.com/conda/conda-build/issues/3166
+    - mkdir -p "$PREFIX/bin"
+    - cp conda-tree.py "$PREFIX/bin/conda-tree"
```

Comment:
This will not work for windows.
Can you make this a python installable package and use the scripts entrypoint?

Reply:
Thanks @mariusvniekerk.  This is not my repo so I was just following @rvalieris's lead but I can now see why that wouldn't work in Windows.

@rvalieris, are you interested in making these changes.  If not, I can make a pull request to your branch if that would be okay.  Much less preferred, I could make a patch here.

Reply:
@rvalieris, I'm going to go ahead and make a PR to your repo.

Reply:
See https://github.com/rvalieris/conda-tree/pull/4

Reply:
just merged the PR, I don't use windows, but feel free to send more PRs if needed.

Reply:
@rvalieris Can you tag a `v0.0.3` release as well?

Reply:
done !